virt/virt-4-9-release-notes.adoc

```diff
@@ -0,0 +1,10 @@
+[id="virt-4-8-release-notes"]
```

Comment:
Incorrect version number. Same comment applies to the context field below as well.